Api/csharp/ArmoniK.Api.Client/Submitter/SubmitterClientExt.cs

```diff
@@ -0,0 +1,284 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2022. All rights reserved.
+//   W. Kirschenmann   <wkirschenmann@aneo.fr>
+//   J. Gurhem         <jgurhem@aneo.fr>
+//   D. Dubuc          <ddubuc@aneo.fr>
+//   L. Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F. Lemaitre       <flemaitre@aneo.fr>
+//   S. Djebbar        <sdjebbar@aneo.fr>
+//   J. Fonseca        <jfonseca@aneo.fr>
+// 
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//         http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+using ArmoniK.Api.gRPC.V1;
+using ArmoniK.Api.gRPC.V1.Submitter;
+
+using Google.Protobuf;
+
+using Grpc.Core;
+
+using JetBrains.Annotations;
+
+namespace ArmoniK.Api.Client.Submitter
+{
+  /// <summary>
+  /// Extension to simplify <see cref="gRPC.V1.Submitter.Submitter.SubmitterClient"/> usage
+  /// </summary>
+  [PublicAPI]
+  public static class SubmitterClientExt
+  {
+    /// <summary>
+    /// Create task request without streaming
+    /// </summary>
+    /// <param name="client">gRPC client to the Submitter</param>
+    /// <param name="sessionId">Id of the sessions</param>
+    /// <param name="taskOptions">Task Options for the tasks in this request</param>
+    /// <param name="taskRequests">The collection of request</param>
+    /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
+    /// <returns>
+    /// The reply to task creation
+    /// </returns>
+    public static async Task<CreateTaskReply> CreateTasksAsync(this gRPC.V1.Submitter.Submitter.SubmitterClient client,
+                                                               string                                           sessionId,
+                                                               TaskOptions?                                     taskOptions,
+                                                               IEnumerable<TaskRequest>                         taskRequests,
```

Comment:
`IAsyncEnumerable`?

Reply:
There is only one reply.  `IAsyncEnumerable` is not needed

Reply:
Making `IAsyncEnumerable` would allow to stream the request and avoids the need to create the array in memory.
But I'm fine to keep `IEnumerable`.

Reply:
My bad, this one is to keep compatibility with the IEnumerable case. I added the IAsyncEnumerable method too.